packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -37,6 +40,12 @@ export default class DefaultPaymentIntegrationService
         );
     }
 
+    createHostedForm(host: string, options: HostedFormOptions): HostedForm {
+        const hostedFormFactory = new HostedFormFactory(this._store);
```

Comment:
Does it make sense to instantiate this in the constructor? Or pass an instance to this class and instantiate that in `create-payment-integration-service.ts` files?

Reply:
I didn't include it in the constructor because 'hostedFormFactory' does not appear to be required all the time and we only need the factory method to create a `HostedForm` for a method.

Reply:
I think it should be ok, since that's how it was passed to the providers before as well. And other dependencies are also passed in the same way to the service. 

Also better not to `new` every time the `createHostedForm` is called. 

cc @davidchin